### DISCUSSION TITLE
＋1️⃣5️⃣ Add Stripe gateway support

### DISCUSSION THREAD

**Discussion by remcotolsma** [pronamic/wp-pronamic-pay#111] [Ideas]
## Links

- https://basecamp.com/1810084/projects/18117394/todos/441785454

**Comment by remcotolsma** [comment]
Re-informed in internal support ticket https://secure.helpscout.net/conversation/1625003045/22691?folderId=1425710.

**Comment by LeoOosterloo** [comment]
New submission from Stay informed (13-10-2021)
--
Product
  | Pronamic Pay
Interest
  | Stripe for WordPress
Language
  | EN

**Comment by LeoOosterloo** [comment]
> I was wondering if you have any indication on the Stripe connection for Pronamic Pay, and if so, if you're maybe looking for a beta testing agency. We'd like to connect an NGO to Stripe for donations, but they lack the resources for building a custom connection by us.

**Comment by rvdsteege** [comment]
With the official Stripe add-on for Gravity Forms, accepting iDEAL payments via **Stripe Checkout** requires the `gform_stripe_session_data` filter to remove/update the `payment_method_types` (defaults to `[ 'card' ]`).

**Comment by rvdsteege** [comment]
Feedback from customer:

> Daar ben ik weer! Wou aangeven dat recurring payments/ subscriptions helaas niet mogelijk is op deze wijze. Eenmalige betalingen gaan prima. Maar zodra je er een subscription van wilt maken moet diegene via sepa debit een reeks informatie invullen wat het onhandig maakt ten opzichte van sepa debit (recurring) via iDEAL.

_Internal Help Scout ticket: https://secure.helpscout.net/conversation/1674637274/22906_

**Comment by rvdsteege** [comment]
From customer:

> Ik kwam hier op: https://www.pronamicpay.com/en/solutions/stripe/, ondersteunt deze de Stripe Checkout (https://stripe.com/nl/payments/checkout) optie? Dat zou fenomenaal zijn. Dan kunnen we dat inzetten voor meerdere landen tegelijk. Mollie ondersteunt alleen wat betaalopties van ons en buurlanden. Wanneer wordt die stripe optie ongeveer verwacht, gezien erbij staat "Available Soon"?

_Internal Help Scout ticket: https://secure.helpscout.net/conversation/2302358014/25904_

**Comment by rvdsteege** [comment]
From customer:

> Klopt het dat jullie een Stripe integratie ook in de nabije toekomst gaan aanbieden via Pronamic Pay (en dat met name voor de Sepa recurring )? Dan lijkt mij dat een betere oplossing dan de directe koppeling WC subscriptions en Stripe.

_Internal Help Scout ticket: https://secure.helpscout.net/conversation/2340421899/26069_

**Comment by rvdsteege** [comment]
From customer (now using Mollie):

> Toppers! En een echt fijne plugin. Redder in nood want ik had al alles ingericht met Stripe en Restrict Content pro. En toen zag ik dat iDEAL incasso niet via Stripe en Restrict Content pro kon.

_Internal Help Scout ticket: https://secure.helpscout.net/conversation/2346243042/26093_

**Comment by remcotolsma** [comment]
Just got a call from Frank (`+316430●●●●●`) asking about this as well, specifically support for Stripe, iDEAL and recurring payments.

**Comment by rvdsteege** [comment]
From customer:

> Idd jammer dat de Recurring Donations op basis van Stripe-add on for GiveWP nog niet werkt met ideal.
>
> Tav van jullie Stripe development activiteiten, hoor daar graag meer over. En als wij als beta-customer deze functie kunnen gebruiken en voor jullie kunnen testen, doen we dat graag!

Recurring donations through iDEAL are not supported in the Recurring Donations add-on for GiveWP:

> **iDeal + Recurring**
> Currently, recurring donations do not work with iDEAL
>
> _Source: https://givewp.com/documentation/add-ons/recurring-donations/supported-payment-gateways/stripe/_

_Internal Help Scout ticket: https://secure.helpscout.net/conversation/2771185689/28051_
